frontend/components/organization/Edit.jsx

```diff
@@ -239,6 +240,26 @@ function Edit({ actions }) {
                       }
                     }
                   />
+                  { member.User.UAAIdentity.origin === 'uaa' && !member.User.signedInAt && (
```

Comment:
This should also include `cloud.gov` as an origin to check.

Reply:
All right; will that literally be the string `'cloud.gov'` in origin?

Reply:
Yes

Reply:
Thanks! Done.